<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>791</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <widget class="QWidget" name="startupMenuPage">
     <widget class="QPushButton" name="connectToBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect To:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="connectToLine">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>601</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="acceptFromBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Acceept From:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="acceptFromLine">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>601</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tryingToConnectPage">
     <widget class="QLabel" name="tryingToConnectLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>167</width>
        <height>25</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Trying to Connect</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="chatPage">
     <widget class="QTextEdit" name="sendText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>711</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>140</y>
        <width>71</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QListWidget" name="chatList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>711</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="exitBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>781</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> Exit Chat</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
